frontend/src/app/store/reducers/cases.ts

```diff
@@ -59,13 +60,13 @@ export const casesSlice = createSlice({
     },
     setCaseFile: (state, action) => {
       const {
-        payload: { note },
+        payload: { note, equipment },
       } = action;
 
       //--
       //-- TODO: need to have each dev add thier state to this section instead of requesting
       //-- each individual state. Add assessment, prevention, equipment here
```

Comment:
This will most likely cause problems the way that its currently written, if note has a value attaches to it and you don't provide a note, it will get set as undefined